app/models/user.rb

```diff
@@ -36,6 +36,16 @@ def core_induction_programme
     mentor_profile.core_induction_programme if mentor?
   end
 
+  def is_on_full_induction_programme?
+    early_career_teacher_profile&.induction_programme_choice == "full_induction_programme" ||
+      mentor_profile&.induction_programme_choice == "full_induction_programme"
+  end
```

Comment:
If you make it an enum on the profiles, you will be able to write
```
early_career_teacher_profile&.full_induction_programme? || mentor_profile&.full_induction_programme?
```
It will also throw an error when it gets a value outside of the enum values, which is a great safeguard against R&P changing their API in the future and silently messing up our database. 

Something like `CourseLessonProgress`'s progress:
```
  enum progress: {
    to_do: "to_do",
    in_progress: "in_progress",
    complete: "complete",
  }
```